charts/k8ssandra/values.yaml

```diff
@@ -122,9 +122,9 @@ cassandra:
 
     # -- Optional datacenter-level heap setting, overrides cluster-level setting
     # `cassandra.heap`
-    heap:
-      size:
-      newGenSize:
+    #heap:
```

Comment:
Same as previous comment.

Reply:
Added additional reference as noted in the commit.

Reply:
See latest comment for impact to this.